frontend/server/src/Controllers/Contest.php

```diff
@@ -1116,7 +1116,7 @@ public static function getContestListDetailsv2ForTypeScript(
         $r->ensureOptionalInt('page_size');
 
         $page = (isset($r['page']) ? intval($r['page']) : 1);
-        $pageSize = (isset($r['page_size']) ? intval($r['page_size']) : 100);
+        $pageSize = (isset($r['page_size']) ? intval($r['page_size']) : 10);
```

Comment:
```suggestion
        $pageSize = (isset($r['page_size']) ? intval($r['page_size']) : 100);
```

hay que dejar esto como estaba para que no se 10x la carga al servidor ^^;;

Reply:
Tenía la idea de que la primera carga del Payload también tenía que ser de 10 Contests. Hay alguna forma de mandarle ese parámetro al momento de cargarlo en el .ts?

Reply:
la primer carga puede ser de 100, eso no hay ningún problema